pyproject.toml

```
@@ -1,3 +1,3 @@
 [tool.black]
 line-length = 120
-target-version = ['py27', 'py36', 'py37', 'py38']
+target-version = ['py36', 'py37', 'py38']
```

```suggestion
target-version = ['py36']
```

also black only needs the lowest version: https://github.com/pytest-dev/pytest/blob/master/pyproject.toml#L106